### DISCUSSION TITLE
Monero donations

### DISCUSSION THREAD

**Discussion by pluja** [photoprism/photoprism#1832] [Q&A] [answered]
I was going to make a Monero donation but crypto donation section has disappeared

**Comment by pluja** [comment, marked as answer]
In #570 crypto donations were accepted and they were available for a quite long time, but today I see they have vanished from the donation options :(

There are addresses on the issue itself but I'm not sure if I should still donate there as funds may be lost if those wallets are no longer under your control.

**Reply by lastzero** [reply]
Nope, here they are: https://github.com/photoprism/photoprism/blob/develop/SPONSORS.md#crypto-wallets

**Reply by pluja** [reply]
Awesome, just donated some monero :)

**Reply by lastzero** [reply]
Thank you! 🎁

**Comment by lastzero** [comment]
## ℹ️ Information on Cryptocurrency Donations Has Been Moved to Our Site ##

Please note that we have moved all the information to our website at **[photoprism.app/crypto-donations](https://photoprism.app/crypto-donations)** to make it more official and provide additional details on why we are doing this, what the differences and benefits are, and what the process looks like.
